subsys/bluetooth/services/ots_olcp_internal.h

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef BT_GATT_OTS_OLCP_H_
+#define BT_GATT_OTS_OLCP_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <sys/types.h>
+#include <zephyr/types.h>
+#include <bluetooth/gatt.h>
+
+/**@brief The types of OLCP procedures. */
```

Comment:
A general minor comment: I think it's customary to have spaces after `/**` and before `*/` in comments